### PR TITLE
Added section to explain OHE and calculate the OHE of training labels.

### PR DESCRIPTION
**Pull request by kxk302**

### CODE REVIEW

**Comment by kxk302** [comment]
The OHE section was missing from the CNN tutorial. Added it in.

**Review by hexylena — APPROVED** [review]

File: topics/statistics/tutorials/CNN/tutorial.md [hunk 1]
```diff
@@ -246,6 +246,29 @@ to compare various Machine Learning techniques.
...
+is 1 (and the rest are 0's); For Orange with label 2, the second element of the array is 1 (and the
+rest are 0's); And for Banana with label 3, the third element of the array is 1 (and the rest are 0's).
+We have 10 digits in our dataset and we would just have an array of size 10, where only one
+element is 1, corresponding to the digit, and the rest are 0's.
+
+### **Create One-Hot Encoding (OHE) representation of training labels**
+
+> ### {% icon hands_on %} Hands-on: One-Hot Encoding
+>
+> - {% tool [To categorical](https://usegalaxy.org/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/sklearn_to_categorical/sklearn_to_categorical/1.0.8.3) %}
```
Comment (hexylena) [inline topics/statistics/tutorials/CNN/tutorial.md:264]:
```suggestion
> - {% tool [To categorical](toolshed.g2.bx.psu.edu/repos/bgruening/sklearn_to_categorical/sklearn_to_categorical/1.0.8.3) %}
```

**Comment by hexylena** [comment]
@anuprulez that suggested change needed to be added first

**Comment by anuprulez** [comment]
@hexylena I will revert it then.

**Comment by hexylena** [comment]
a second PR making that change would be fine too, easier than reverting

**Comment by anuprulez** [comment]
Can you merge this PR that reverts the changes https://github.com/galaxyproject/training-material/pull/3242
@hexylena Thank you!

**Comment by hexylena** [comment]
merged.

**Comment by hexylena** [comment]
@anuprulez will you open a new pr then with @kxk302's changes?

**Comment by anuprulez** [comment]
yes sure!

**Comment by anuprulez** [comment]
@hexylena this should work https://github.com/galaxyproject/training-material/pull/3243. It has the fix for the suggested change in the old PR

**Pull request merged**